craft_cli/messages.py

```diff
@@ -38,7 +38,7 @@
 import platformdirs
 
 try:
-    import win32pipe
+    import win32pipe  # type: ignore[import]
```

Comment:
I removed the previous `# type: ignore` comment because ruff complains that it's too general, and then re-added this specific one when either mypy or pyright complained about the import